src/main/resources/assets/farmersdelight/lang/es_mx.json

```diff
@@ -274,7 +274,7 @@
   "farmersdelight.block.skillet.invalid_item": "Esto no se puede cocinar...",
   "farmersdelight.item.skillet.how_to_cook": "¡Usa tu otra mano para sostener ingredientes!",
 
-  "death.attack.farmersdelight.stove": "%1$s fue cocinado a la perfección",
+  "death.attack.farmersdelight.stove": "%1$s Fue cocinado a la perfección",
```

Comment:
This is not impeditive, just a curiosity:
The `%1$s` represents the victim's name, making it a sentence. Wouldn't "fue" be lower-case here? Or is that how death messages are written in es_mx?

Reply:
No, youre right, they mean to be lower-case. I did'nt remember why i changed that exact sentence. let me revert that.

Reply:
Done:)